.github/workflows/ci.yml

```diff
@@ -52,55 +37,23 @@ jobs:
 
   tests:
     name: Tests
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        build: [stable, beta]
-        include:
-          - build: stable
-            os: ubuntu-latest
-            rust: stable
-          - build: beta
-            os: ubuntu-latest
-            rust: beta
+    runs-on: ubuntu-latest
+    container: kisiodigital/rust-ci:latest-proj
     steps:
-    - name: Install proj
-      run: |
-        wget --quiet --output-document - "https://kisiodigital.jfrog.io/kisiodigital/api/gpg/key/public" | sudo apt-key add -
-        echo "deb [arch=amd64] https://kisiodigital.jfrog.io/kisiodigital/debian-local stretch main" | sudo tee /etc/apt/sources.list.d/kisio-digital.list
-        sudo apt update
-        sudo apt install --yes libxml2-utils pkg-config libssl-dev clang libtiff-dev libcurl4-nss-dev proj=${PROJ_VERSION}
```

Comment:
Oops, `libxml2-utils` is needed for tests (and `xmllint` verification). See 349c181066715a14f1e4714eb5bd3f1ae5d65e9f.